### DISCUSSION TITLE
Custom fixture cannot be used

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2696] [Q&A] [answered]
<img width="613" alt="image" src="https://github.com/seleniumbase/SeleniumBase/assets/32641509/41b4892c-4525-4bd3-9089-9f394f167f5d">

In this way, login fixture cannot obtain
Inheriting the BaseCase class is running the project in unittest mode

**Comment by mdmintz** [comment, marked as answer]
See https://github.com/seleniumbase/SeleniumBase/issues/745#issuecomment-738082454

That's the main reason why the ``sb`` fixture was created. (Example: [SeleniumBase/examples/test_sb_fixture.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_sb_fixture.py))

Most pytest fixtures have an incompatibility with ``unittest.TestCase``, which SeleniumBase's `BaseCase` inherits. Use one of the `sb` `pytest` fixture formats from [SeleniumBase/help_docs/syntax_formats.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md). It allows you to use other pytest fixtures in your tests.
